src/pages/Animeworld/main.ts

```diff
@@ -21,6 +21,17 @@ export const Animeworld: pageInterface = {
     getOverviewUrl(url) {
       return `${Animeworld.domain}/${Animeworld.sync.getIdentifier(url)}`;
     },
+    getMalUrl(provider) {
+      return new Promise(resolve => {
+        if (provider === 'ANILIST') {
+          resolve(j.$('#mal-button').attr('href') || false);
+        }
+        if (provider === 'MAL') {
```

Comment:
it should always return mal if its available like on the other implementations

Reply:
Understood 👍